content/guides/kubernetes/what-is-velero.md

```diff
@@ -46,4 +46,4 @@ Data protection is a chief concern for application owners who want to make sure
 
 On the [Velero home page](https://velero.io/) you can get information on the latest release and download Velero from Github. 
 
-To get started using Velero watch these videos covering two of Velero’s useful features, [Backup and Restore,](https://kube.academy/lessons/backup-restore) and [Migration](https://www.youtube.com/watch?v=q2FCxheA8VI&list=PL7bmigfV0EqQRysvqvqOtRNk4L5S7uqwM&index=5&t=0s).
\ No newline at end of file
+To get started using Velero read our guide, [Getting Started with Velero](/guides/kubernetes/velero-gs.md), and watch these videos covering two of Velero’s useful features, [Backup and Restore,](https://kube.academy/lessons/backup-restore) and [Migration](https://www.youtube.com/watch?v=q2FCxheA8VI&list=PL7bmigfV0EqQRysvqvqOtRNk4L5S7uqwM&index=5&t=0s).
```

Comment:
```suggestion
To get started using Velero read our guide, [Getting Started with Velero](/guides/kubernetes/velero-gs), and watch these videos covering two of Velero’s useful features, [Backup and Restore,](https://kube.academy/lessons/backup-restore) and [Migration](https://www.youtube.com/watch?v=q2FCxheA8VI&list=PL7bmigfV0EqQRysvqvqOtRNk4L5S7uqwM&index=5&t=0s).
```

Reply:
Thanks and whoops I pushed to the same PR/branch...